File: new-backend/api/internal/models/voting_round.go
package models

import "time"

// VotingRound is the DB document. Name is kept equal to Title for frontend compatibility.
type VotingRound struct {
	ID          string    `bson:"_id"         json:"id"`
	VoteId      int       `bson:"voteId"      json:"voteId"`
	Title       string    `bson:"title"       json:"title"`
	Name        string    `bson:"name"        json:"name"`
	Description string    `bson:"description" json:"description"`
	VoteDate    time.Time `bson:"voteDate"    json:"voteDate"`
}

File: Backend/api/app/schemas/voting_round.py
from uuid import UUID
from datetime import datetime
from pydantic import BaseModel


class RoundSchema(BaseModel):
    Id: UUID
    Name: str
    Title: str
    Description: str = ""
    VoteDate: datetime
    VoteId: int = 0

    model_config = {"from_attributes": True}
